tendrl/monitoring_integration/sync/__init__.py

```diff
@@ -11,6 +16,28 @@ def __init__(self):
         self._complete = gevent.event.Event()
 
     def _run(self):
-        while not self._complete.is_set():
-            webhook_receiver.start_server()
+        interval = etcd_utils.read("_NS/gluster/config/data/sync_interval")
+
+        try:
 
+            interval = float(interval.value)
+
+        except ValueError as ex:
+            logger.log("error", NS.get("publisher_id", None),
+                       {'message': str(ex)})
+            raise ex
+
+        while not self._complete.is_set():
+            try:
+                gevent.sleep(interval)
+                plugin_obj = GraphitePlugin()
+                objects  = NS.monitoring.definitions.get_parsed_defs()["namespace.monitoring"]["graphite_data"]
```

Comment:
These "objects" arent going to change since we dont yet allow updates to definitions of a live Tendrl cluster, so please move "objects = .."  outside the while loop